wikipediapreview-wordpress.php

```diff
@@ -1,6 +1,13 @@
 <?php
 /**
  * Plugin Name: Wikipedia Preview
- * Description: Add Wikipedia Preview to your articles
- * Version: 0.1.0
+ * Plugin URI: https://github.com/wikimedia/wikipedia-preview
```

Comment:
this shows like this, the URI should be `https://wordpress.org/plugins/wikipedia-preview/` if we have this domain in wordpress :)
![image](https://user-images.githubusercontent.com/2560096/116459822-90899b80-a866-11eb-82f8-e3ba7a602937.png)

Reply:
[Doc says](https://developer.wordpress.org/plugins/plugin-basics/header-requirements/) we can't use a wordpress.org url:

> Plugin URI: The home page of the plugin, which should be a unique URL, preferably on your own website. This must be unique to your plugin. You cannot use a WordPress.org URL here.

I suppose this would be a good place for the landing page when ready though: https://phabricator.wikimedia.org/T274136

Reply:
You are right! Let's keep what it is right now.